tests/pages/owner/non-existent.js

```diff
@@ -10,6 +10,6 @@ export default create({
   visit: visitable('/non-existent-owner'),
   showsBarricadeIllustration: contains('svg', { scope: '.page-graphic' }),
   errorMessage: text('.missing-notice .page-title'),
-  errorMessageProisHidden: isHidden('.missing-notice p'),
+  errorMessageProIsHidden: isHidden('.missing-notice p'),
```

Comment:
Depending on how you do things, this could be an API change. I'm assuming it isn't.

I was also guessing about what it was trying to say, but given that this is `isHidden`, I'm vaguely confident I'm right.